packages/orchestrator-ui-components/src/components/WfoSubscriptionsList/WfoSubscriptionsList.tsx

```diff
@@ -69,91 +70,88 @@ export const WfoSubscriptionsList: FC<WfoSubscriptionsListProps> = ({
-    // https://github.com/workfloworchestrator/orchestrator-ui/issues/262
-
     const router = useRouter();
     const t = useTranslations('subscriptions.index');
     const tError = useTranslations('errors');
     const { showToastMessage } = useShowToastMessage();
 
-    const tableColumns: WfoTableColumns<SubscriptionListItem> = {
-        subscriptionId: {
-            field: 'subscriptionId',
-            name: t('id'),
-            width: '100',
-            render: (value) => <WfoFirstPartUUID UUID={value} />,
-            renderDetails: (value) => value,
-        },
-        description: {
-            field: 'description',
-            name: t('description'),
-            width: '400',
-            render: (value, record) => (
-                <Link href={`/subscriptions/${record.subscriptionId}`}>
-                    {value}
-                </Link>
-            ),
-        },
-        status: {
-            field: 'status',
-            name: t('status'),
-            width: '110',
-            render: (value) => <WfoSubscriptionStatusBadge status={value} />,
-        },
-        insync: {
-            field: 'insync',
-            name: t('insync'),
-            width: '110',
-            render: (value) => <WfoInsyncIcon inSync={value} />,
-        },
-        productName: {
-            field: 'productName',
-            name: t('product'),
-        },
-        tag: {
-            field: 'tag',
-            name: t('tag'),
-            width: '100',
-        },
-        customerFullname: {
-            field: 'customerFullname',
-            name: t('customerFullname'),
-            width: '150',
-        },
-        customerShortcode: {
-            field: 'customerShortcode',
-            name: t('customerShortcode'),
-            width: '150',
-        },
-        startDate: {
-            field: 'startDate',
-            name: t('startDate'),
-            width: '150',
-            render: (value) => <WfoDateTime dateOrIsoString={value} />,
-            renderDetails: parseDateToLocaleDateTimeString,
-            clipboardText: parseDateToLocaleDateTimeString,
-        },
-        endDate: {
-            field: 'endDate',
-            name: t('endDate'),
-            width: '150',
-            render: (value) => <WfoDateTime dateOrIsoString={value} />,
-            renderDetails: parseDateToLocaleDateTimeString,
-            clipboardText: parseDateToLocaleDateTimeString,
-        },
-        note: {
-            field: 'note',
-            name: t('note'),
-        },
-        metadata: {
-            field: 'metadata',
-            name: t('metadata'),
-            render: (value) => <WfoInlineJson data={value} />,
-            renderDetails: (value) =>
-                value && <WfoJsonCodeBlock data={value} isBasicStyle />,
-        },
-    };
+    const tableColumnConfig: WfoAdvancedTableColumnConfig<SubscriptionListItem> =
+        {
+            subscriptionId: {
+                columnType: ColumnType.DATA,
```

Comment:
This seems a bit repeated. What about making the default  ColumnType = DATA ?

Reply:
In the type `WfoAdvancedTableColumnConfig` the `columnType` prop is used to discriminate between the 2 types. As of now it does not seem to be possible to make it optional. Making it optional allowes both props renderData and renderControl to be part of the configuartion